frontend/styles/_settings.scss

```diff
@@ -80,6 +80,8 @@ $light-gray: #f6f6f6;
 $mid-light-gray: #e5e5e5;
 $medium-gray: #dddddd;
 $gray-text: #515253;
+$yellow-gray: #d9d9d9;
+$yet-another-gray: #999999;
```

Comment:
😆😆😆 
Maybe something more descriptive? Although we have a lot it's true. Is the only one we will remember

Reply:
I'm already at the point where I need to search which gray in the design equals which variable. I'm not sure it's worth it